modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java

```diff
@@ -162,6 +162,20 @@ public ObjectDefinition getObjectDefinition() {
 			ObjectWebKeys.OBJECT_DEFINITION);
 	}
 
+	public ObjectField getObjectFieldbyName(
```

Comment:
I think we should refactor this method name to getObjectFieldByName to follow the camel-case naming convention.

Reply:
Thank you, I updated code